manifest.samples.json

```diff
@@ -57,9 +57,12 @@
             {
               "dockerfile": "samples/dotnetapp",
               "os": "windows",
-              "osVersion": "windowsservercore-2009",
+              "osVersion": "windowsservercore-20H2",
               "tags": {
-                "dotnetapp-windowsservercore-2009": {}
+                "dotnetapp-windowsservercore-20H2": {},
+                "dotnetapp-windowsservercore-2009": {
```

Comment:
The same comment I made in the .NET Core change applies here.  I question the need to maintain the 2009 tags in the samples repo.

Reply:
Fixed